modules/motions/Snapshot.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file   Snapshot.h
+ * @brief  写真を撮影して保存するクラス
+ * @author takuchi17
+ */
+
+#ifndef SNAPSHOT_H
+#define SNAPSHOT_H
+
+#include "Motion.h"
+
+class Snapshot : public Motion {
+ public:
+  /**
+   * コンストラクタ
+   * @param _robot ロボットインスタンス
+   * @param _robot ファイル名
+   */
+  Snapshot(Robot& _robot, const std::string& _fileName);
```

Comment:
デフォルト値があってくれた方がうれしみがつよい

```suggestion
   * @param _robot ファイル名 (デフォルト: snapshot.JPEG)
   */
  Snapshot(Robot& _robot, const std::string& _fileName = "snapshot");
```